transcom/mymove
# B 20079 int

## [B-20079](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A950887)

## Summary

Change the Review weights page label for "Weight requested" to "Actual weight".

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Login as a TIO
2. Select a shipment
3. Click the Review Weights button
4. Verify "Actual weight" label is displayed 
![image](https://github.com/transcom/mymove/assets/149000222/4845bd95-a08a-4578-a004-5bbafa784bd2)


### Frontend
